Composer/packages/client/src/pages/notifications/useNotifications.tsx

```diff
@@ -13,7 +14,7 @@ import { getReferredFiles } from './../../utils/luUtil';
 export default function useNotifications(filter?: string) {
   const { state } = useContext(StoreContext);
   const { dialogs, luFiles, lgFiles } = state;
-
+  const inLineLgTemplateFormat = /^bfd(activity|prompt|defaultValueResponse|invalidPrompt|unrecognizedPrompt)-[0-9]+/;
```

Comment:
you can leverage on this existed pattern
https://github.com/microsoft/BotFramework-Composer/blob/c58666eafaeb46d6be511431cc46e7c3c0764ec7/Composer/packages/lib/shared/src/lgUtils/parsers/lgPatterns.ts#L4-L6